books/serializers.py

```
@@ -107,30 +106,18 @@ class Meta:
             "date_added",
-            "date_started",
-            "date_finished",
             # Book creation fields
             "title",
             "author_name",
             "genre",
             "tagline",
             "description",
         ]
-        read_only_fields = ["date_added", "date_started", "date_finished"]
+        read_only_fields = ["date_added"]
```

Ruff (0.13.1)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)